### PR TITLE
Make axa text variants mutually exclusive

### LINKED ISSUE #1587: Documentation of the Atom AXA Text is incomplete
In der Doku vom AXA Text fehlt der Hinweis das man Varianten kombinieren kann. 
zum Beispiel:
`<AXATextReact variant={"bold size-3"}>...` 
währe toll wenn man die Doku erweitern könnte

Chatverlauf mit Luca Mele:

Lars Volkheimer 9:39 AM
hey luca ich ha e luge i de doku vom axatext gfunde mer cha ja au variant="bold size-3" age zum bispiel.
Luca Mele:speech_balloon: 9:40 AM
Meinst du dass es möglich ist 2  varianten zu kombinieren? (edited) 
Lars Volkheimer 9:41 AM
ja im code sind die varianten mit include abgefragt und es funktioiniert auch
ich habs ausprobiert
Luca Mele:speech_balloon: 9:41 AM
sehr gut. Willst ein github issue machen? das sollte nicht sein :slightly_smiling_face:
danke dir vielmals
Lars Volkheimer 9:42 AM
ja kann ich machen

### PR DESCRIPTION
**Pull request by raphaellueckl**
Fixes #1587

# Done is when (DoD):
- [x] I have added UI/Unit tests for my changes
- [x] Old tests and eslint rule are not broken
- [x] I added modified component properties to the typescript typings
- [ ] Design has been reviewed with (here name of reviewer)
- [x] My changes are accordingly to our [Best Practices](https://github.com/axa-ch/patterns-library/blob/develop/CONTRIBUTION.md#best-practices)
- [x] My changes are well documented in the README and showcased in the stories 
- [x] My changes are shortly added to CHANGELOG.md with the issue number
- [x] Tested with IE

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas (focus on **WHY**)
- [x] I have made/updated the ChangeLog Section in the README

### CODE REVIEW

File: src/components/10-atoms/text/ui.test.js [hunk 1]
```diff
@@ -141,21 +143,45 @@ test('should have correct font definitions for text size 2 with custom span tag'
...
-
 fixture('Text - Bold')
   .page(`${host}/iframe.html?id=atoms-text--text&knob-variant=bold`)
   .beforeEach(async t => {
     await t.resizeWindow(380, 680);
   });
 
 test('should have correct font weight for text bold', async t => {
+  const $axaElemShadow = await Selector(() =>
+    document.querySelector('axa-text[variant="bold"]').querySelector('.a-text')
```
Comment (01231) [inline src/components/10-atoms/text/ui.test.js]: I would make a selector with the two selectors combined

Comment (raphaellueckl) [reply src/components/10-atoms/text/ui.test.js]: I did not touch this code, that is git having that part messed up.

**Review by 01231 — DISMISSED** [review]

File: src/components/10-atoms/text/ui.test.js (hunk 1 above)
Comment (LucaMele) [reply src/components/10-atoms/text/ui.test.js]: I would still do it anyway. is a small thing. and i recon is prettier, not git.

**Review by LucaMele — DISMISSED** [review]
LGTM. Just the nice to have that noah said but no merge blocker for me

**Review by 01231 — DISMISSED** [review]

File: src/components/10-atoms/text/ui.test.js (hunk 1 above)
Comment (raphaellueckl) [reply src/components/10-atoms/text/ui.test.js]: Aha, I misunderstood.

**Review by 01231 — APPROVED** [review]

**Pull request merged**